tests/serving/test_tracking.py

```diff
@@ -28,6 +28,7 @@
     register_temporary_client_datastore_profile,
     remove_temporary_client_datastore_profile,
 )
+from mlrun.model_monitoring.applications import ModelMonitoringApplicationBase
```

Comment:
It's not the appropriate place for this test, as it's not related to serving/tracking.
How about `tests/model_monitoring/test_applications/test_base.py`?